tests/utils/test_helpers.py

```diff
@@ -175,6 +177,43 @@ def test_extend_hub_uri():
             assert expected_output == output
 
 
+def test_validate_tag_name():
+    cases = [
+        {
+            "tag_name": "tag_name",
+            "raise": True,
```

Comment:
```suggestion
            "raise_on_failure": True,
```
Otherwise I initially thought this was an expectation, not a parameter